pennylane/templates/subroutines/adder.py

```diff
@@ -22,54 +22,75 @@
 class Adder(Operation):
     r"""Performs the in-place modular addition operation.
 
-     This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
-     computational basis:
+    This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
+    computational basis:
 
-     .. math::
+    .. math::
 
-         \text{Adder}(k, mod) |x \rangle = | x+k \; \text{modulo} \; mod \rangle.
+        \text{Adder}(k, mod) |x \rangle = | x+k \; \text{mod} \; mod \rangle.
 
-     The implementation is based on the quantum Fourier transform method presented in
-     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
+    The implementation is based on the quantum Fourier transform method presented in
+    `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
 
     .. note::
 
-        Note that :math:`x` must be smaller than :math:`mod` to get the correct result.
+        To obtain the correct result, :math:`x` must be smaller than :math:`mod`.
 
+    .. seealso:: :class:`~.PhaseAdder` and :class:`~.OutAdder`.
 
-     Args:
-         k (int): the number that needs to be added
-         x_wires (Sequence[int]): the wires the operation acts on
-         mod (int): the modulus for performing the addition, default value is :math:`2^{len(x\_wires)}`
-         work_wires (Sequence[int]): the auxiliary wires to be used for performing the addition
+    Args:
+        k (int): the number that needs to be added
+        x_wires (Sequence[int]): the wires the operation acts on
+        mod (int): the modulo for performing the addition. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`.
```

Comment:
```suggestion
        mod (int): the modulo for performing the addition. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`
```